### PR TITLE
[FEAT] Swipe Page Styling

### LINKED ISSUE #298: [FEAT] Modify the styling for frontend components
## Feature Description: 
- Modify the styling of the frontend components so that the app has a dark theme and a cleaner aesthetic
- Remove the Comic sans fonts and the cartoon aesthetic
- Make use of Bootstrap components for layout and styling

The changes will be made in frontend/components

## Blocked by:
#17

### PR DESCRIPTION
**Pull request by reeved**
Helps fix #298.

- Restyled Swiping Page and added Dark Theme to the Swipe Cards
- Darkened the background image for better text visibility
- Fixed a bug in Sockets when a user would disconnect, crashing the server
- Restyled Help Modals with Dark Theme
- Minor tweaks to other pages

**Note:** a quirk of adding dark theme to the Modals is that the Pie Chart legend has its legibility decreased. I tried different methods to change its font colour but was not able to do so. It's not an app-breaking bug so if someone has time to look into a way to change the font colour, feel free to :)

Before:
image
image

After:
image
image

### CODE REVIEW

File: frontend/src/components/7.Result/ResultPage.js [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
-const price = "$$$";
+const name = 'Lonestar';
+const location = 'Botany';
+const price = '$$$';
 const rating = 4.0;
 
 /**
  * @param {*} props
  * @return {*}
  * TODO: remove hard-coded location for the winning restaurant coordinates
```
Comment (Dhors) [inline frontend/src/components/7.Result/ResultPage.js:30]: this was left in from the older version before I added my comment,
What should we do about this?

**Review by Dhors — APPROVED** [review]
Looked through, nice changes to the styling and added consistency in the codebase reeve!

Sorry about the ESLint errors, I'll be sure to check the code more properly.

File: frontend/src/components/7.Result/ResultPage.js (hunk 1 above)
Comment (reeved) [reply frontend/src/components/7.Result/ResultPage.js:30]: good catch! I've updated the file to remove this comment and another unneeded one.

**Review by SreenikethRaghavan — APPROVED** [review]
LGTM! Great work on the styling @reeved! 👍

**Pull request merged**